logger_registry.go

```diff
@@ -0,0 +1,109 @@
+package requestcontext
+
+import (
+	"fmt"
+	"sync"
+
+	"github.com/op/go-logging"
+)
+
+// LoggerRegistry provides the api for a registry of loggers.
+type LoggerRegistry interface {
+	MustCreate(name string, level ...string) Logger
```

Comment:
what is the purpose of these levels? The provided implementation only allows 1 parameter